src/components/book/AddBook.js

```diff
@@ -1,12 +1,44 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/Books';
+import {
+  FormContainer,
+  AddBookInput,
+  CategoryInput,
+  FormWrapper,
+  Headings,
+  Button,
+} from './AddBookStyles';
 
-const AddBook = () => (
-  <div>
-    <div className="add-book">
-      <input type="text" placeholder="Add Book" className="add-book-input" />
-      <button type="submit">Add</button>
-    </div>
-  </div>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: uuidv4(),
+      title: e.target.title.value,
+      author: e.target.author.value,
+
+    };
+
+    dispatch(addBook(newBook));
+    e.target.title.value = '';
+    e.target.author.value = '';
+  };
+
+  return (
+    <FormWrapper id="add-books">
+      <Headings>ADD NEW BOOK</Headings>
+      <FormContainer onSubmit={submitBookToStore}>
+        <AddBookInput placeholder="Book title" name="title" />
+        <AddBookInput placeholder="Book author" name="author" />
+        <CategoryInput placeholder="Not available" />
+        <Button type="submit">ADD BOOK</Button>
+      </FormContainer>
+    </FormWrapper>
```

Comment:
- [ ] Your inputs should be controlled and states should be the only source of truth. You can refer to the first paragraph of this [article](https://reactjs.org/docs/forms.html) to get an idea of what this means.

  As listed in the project requirements (see image below) your add book form component should have a local `state` for `title` and `author`. And that local state should be updated whenever a user updates the title and author inputs with `onchange`. This state is what should be dispatched. You can refer to this [article](https://dmitripavlutin.com/controlled-inputs-using-react-hooks/) for a quick understanding on how to create controlled components with React hooks.

  ![muth-req](https://user-images.githubusercontent.com/36542986/146150850-cbd4e1b0-84a1-448b-8227-102f4459f7fe.jpg)

Reply:
Thanks, @TedLivist. recommended changes fixed.